converter/dicom-cast/src/Microsoft.Health.DicomCast.Hosting/Program.cs

```diff
@@ -45,30 +48,30 @@ public static void Main(string[] args)
 
                 services.AddHostedService<DicomCastBackgroundService>();
 
-                AddApplicationInsightsTelemetry(services, configuration);
+                AddOpenTelemetry(services, configuration);
             })
             .Build();
 
         host.Run();
     }
 
     /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
+    /// Adds Open telemetry exporter for Azure monitor.
     /// </summary>
-    private static void AddApplicationInsightsTelemetry(IServiceCollection services, IConfiguration configuration)
+    private static void AddOpenTelemetry(IServiceCollection services, IConfiguration configuration)
     {
+        services.Add<DicomCastMeter>();
```

Comment:
Is this singleton, scoped or Transient? Can you make it explicit. Looks like it needs to be singleton

Reply:
Thank you Smitha, yes it should be singleton. I have made the changes.